### ISSUE TITLE
Improvement: Add ability to call an external shell + Terminal icon to the toolbar

### ISSUE THREAD

**Issue by luzpaz** [francescmm/GitQlient#244]
**What current behavior doesn't fully fit your needs? Why?**
The interface doesn't provide a clear way to call an external shell 
![Screenshot_20220402_124402](https://user-images.githubusercontent.com/4140247/161392997-96d23653-21ae-4e57-a036-c732f573ce36.png)

**What you'd like to see? Propose a solution**
![image](https://user-images.githubusercontent.com/4140247/161393027-ee3d58bf-f7c0-4875-9045-b37a328ed941.png)
This is a quick mockup but this could rapidly help users orient to the UI and grok that this button calls a terminal

**Describe alternatives you've considered**
Just try to find it. I can't find

**Additional context**
It may be also useful to offer the option in the burger menu icon in the far top left corner 
![Screenshot_20220402_125457](https://user-images.githubusercontent.com/4140247/161393223-e5d3be83-9990-49fe-b8e4-d00493eeee91.png)

**Comment by francescmm** [comment]
This is kinda funny. In the first version of GitQlient I actually had a terminal to execute Git commands, but I discarded since people were not that eager to open it. In fact, the original icon is still there. I saw that GitKraken include one in some release as well.

I guess you'd like to have the same auto-complete behaviour as in a normal terminal, right?

**Comment by luzpaz** [comment]
auto-complete is pretty nice to have with a terminal :innocent:

**Comment by francescmm** [comment]
I'll check what's the cost of adding a terminal widget in GitQlient, I know there are some plugins although this might make the app grow in size a bit :)

**Comment by francescmm** [comment]
The changes are ready to be merged in the original repo (lxqt/qtermwidget repo), but since they aren't yet merged and they won't be part of the release until at least November, I've decided to fork the repo to add support in GitQlient until the original repo is ready.

This will be part of GitQlient 1.5 but due to the restrictions of qtermwidget, it will be available on MacOS and Linux until further notice.

**Comment by luzpaz** [comment]
Thank you for all you work and effort!

**Comment by luzpaz** [comment]
:tada:

**Comment by band-a-prend** [comment]
I tried to download plugin but with it GitQlient doesn't launched:
```
libQTermWidget.so: undefined symbol: _ZdlPvm, version Qt_5
```

But I could confirm that after that if I install system `x11-libs/qtermwidget-1.2.0` (Gentoo Linux) and create symlink to `libqtermwidget5.so.1.2.0` with name `libQTermWidget.so` the plugin successfully activated and after enabling it I could switch to terminal. But I don't know how to change background of it: it's grey but it works so it's not critical - at least I could use git push via ssh now!

If I set `/usr/lib64` as folder of plugin then it's significantly increase launch time of GitQlient. So I prefered to use symlink in HOME directory subfolder.
